main.py

```diff
@@ -1,7 +1,23 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    actors = [("George", "Klooney"), ("Kianu", "Reaves"),
```

Comment:
as for me it can looks better if you rewrite each name from new line 
https://peps.python.org/pep-0008/#indentation